wordpress-mobile/WordPress-Android
# Use previous media picker in stories

We've decided to not use the consolidated media picker for the Stories project. This PR disables it (instead of relying on the flag) but keeps the functionality in place in case we want to go back. 

As part of this project I'm moving the FAB in the old photo picker up so it doesn't overlap the bottom bar. The FAB was previously used only in the stories so there shouldn't be any issue with moving it in the layout. 

To test:
- Turn on the Consolidated picker flag in App Settings/Test feature config
- Go to My Site/Add Story
- Notice the previous UI is visible
- Notice the FAB is moved up

![Screenshot_1604480715](https://user-images.githubusercontent.com/1079756/98094555-bde22780-1e89-11eb-8a66-01ee806ca13f.png)

PR submission checklist:
